bc_obps/registration/tests/endpoints/v2/test_operations.py

```diff
@@ -17,115 +15,49 @@
 
 
 class TestOperationsEndpoint(CommonTestSetup):
-    endpoint = custom_reverse_lazy("list_operations")
-
-    def test_operations_endpoint_list_operations_v2_paginated(self):
-        operator1 = operator_baker()
-        baker.make_recipe(
-            'utils.operation',
-            operator_id=operator1.id,
-            status=Operation.Statuses.PENDING,
-            _quantity=60,
-        )
-        # Get the default page 1 response
-        response = TestUtils.mock_get_with_auth_role(self, "cas_admin")
-        assert response.status_code == 200
-        response_data = response.json().get('items')
-        # save the id of the first paginated response item
-        page_1_response_id = response_data[0].get('id')
-        assert len(response_data) == PAGE_SIZE
-        # Get the page 2 response
-        response = TestUtils.mock_get_with_auth_role(
-            self,
-            "cas_admin",
-            self.endpoint + "?page=2&sort_field=created_at&sort_order=desc",
-        )
-        assert response.status_code == 200
-        response_data = response.json().get('items')
-        # save the id of the first paginated response item
-        page_2_response_id = response_data[0].get('id')
-        assert len(response_data) == PAGE_SIZE
-        # assert that the first item in the page 1 response is not the same as the first item in the page 2 response
-        assert page_1_response_id != page_2_response_id
-
-        # Get the page 2 response but with a different sort order
+    @patch("service.operation_service_v2.OperationServiceV2.list_operations_timeline", autospec=True)
```

Comment:
The old test tested pagination and filtering here. I tested that in the service tests instead